politica-w-ibid.csl

```diff
@@ -0,0 +1,351 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" demote-non-dropping-particle="sort-only" default-locale="da-DK">
+  <info>
+    <title>Politica m. ibid (Dansk)</title>
+    <title-short>Politica m. ibid</title-short>
+    <id>http://www.zotero.org/styles/politica-w-ibid</id>
+    <link href="http://www.zotero.org/styles/politica-w-ibid" rel="self"/>
+    <link href="https://politica.dk/fileadmin/politica/Politica_skrivevejledning.pdf" rel="documentation"/>
+    <author>
+      <name>Aleksander Bang-Larsen</name>
+      <email>github@aleksanderbl.dk</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="political_science"/>
+    <updated>2025-08-31T15:00:00+01:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="da-DK">
+    <terms>
+      <term name="editor" form="short">red.</term>
```

Comment:
This term would already be translated like this for you, so adding this override here is not necessary.
Can you check agains the Danish locale and remove any such terms here?
See https://github.com/citation-style-language/locales/blob/master/locales-da-DK.xml